api/client/proto/authservice.proto

```diff
@@ -917,20 +917,15 @@ message DeleteMFADeviceResponseAck {}
 
 // DeleteMFADeviceSyncRequest is a request to delete a MFA device (nonstream).
 message DeleteMFADeviceSyncRequest {
-    // TokenID is the ID of a user token that will be used to verify this request.
-    // Token types accepted are:
-    //   - Recovery approved token that is obtained with RPC ApproveAccountRecovery
-    //   - Privilege token that is obtained with RPC CreatePrivilegeToken
-    string TokenID = 1 [ (gogoproto.jsontag) = "token_id" ];
+    // PrivilegeTokenID is the ID of a privilege token that will be used to verify this request.
+    string PrivilegeTokenID = 1 [ (gogoproto.jsontag) = "privilege_token_id" ];
```

Comment:
Do you think it's worth renaming this one? I'm thinking we should keep the more generic "Token ID" name used throughout the API (with supporting documentation), so as we add/remove support for other tokens we keep the nomenclature consistent.

A possible exception would be places where we are absolutely sure that only a single token type will ever be accepted (RPCs exclusive to the account recovery flow, for example).

Applies elsewhere as well.